OpenEdXMobile/src/main/java/org/edx/mobile/social/facebook/GetUserCallback.java

```diff
@@ -0,0 +1,59 @@
+package org.edx.mobile.social.facebook;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import com.facebook.GraphRequest;
+import com.facebook.GraphResponse;
+
+import org.edx.mobile.social.SocialMember;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class GetUserCallback {
+    public interface IGetUserResponse {
```

Comment:
This isn't a convention that we should follow for interfaces. We do have legacy interfaces based on this convention but it should be avoided as its not an Android-related convention.